src/homePage.js

```diff
@@ -0,0 +1,58 @@
+const homePage = () => {
+  const homeContainer = document.createElement('div');
+
+  // const heading = document.createElement("h2");
+  // heading.className = "text-center";
+  // heading.innerText = "Tanzila's Cafe";
+  // homeContainer.appendChild(heading)
```

Comment:
- [ ] It will be great to delete all the commented code in your project. This will keep your work clean and professional. Remember, you can always use `git history` to go back to the previous version of your application. ⬇️ 
<img width="792" alt="no_inline_comments" src="https://user-images.githubusercontent.com/32361963/127464972-1b3c6a40-976e-4bd5-925f-cf463fec186e.png">